### ISSUE TITLE
Create a documentation page about how to build a discord bot against our inference backend

### ISSUE THREAD

**Issue by yk** [LAION-AI/Open-Assistant#2340]
- analyze what the web-frontend is doing at the /chat route
- translate that into instructions on how to potentially set up a discord bot

**Comment by shreesha345** [comment]
can I work on this issue

**Comment by olliestanley** [comment]
@shreesha345 Sure! I have assigned you

**Comment by shreesha345** [comment]
thank you I will do that

**Comment by shreesha345** [comment]
Now I am confused with some things like where is the Chat response coming from or is there any api?

**Comment by olliestanley** [comment]
The inference backend code (Python/FastAPI) being called by the frontend can be found [here](https://github.com/LAION-AI/Open-Assistant/tree/main/inference/server/oasst_inference_server/routes). I'm not sure if we have API docs deployed anywhere at the moment

**Comment by shreesha345** [comment]
ok let me check that out

**Comment by yk** [comment]
hi @shreesha345 do you have updates on this?

**Comment by shreesha345** [comment]
Yes I am working on the api provides by huggingface so that it will be easy to integrate to discord

**Comment by shreesha345** [comment]
Because I have no clue of the api infrastructure which you provided me so I am using huggingface

**Comment by wirthual** [comment]
Hi, 

@shreesha345  I found the hosted API description here: https://projects.laion.ai/Open-Assistant/api
